python/paddle/incubate/distributed/models/moe/moe_layer.py

```diff
@@ -314,6 +314,8 @@ def __init__(self,
         super(MoELayer, self).__init__()
 
         recompute_interval = kwargs.get("recompute_interval", 0)
+        recompute_offload = kwargs.get("recompute_offload", False)
```

Comment:
Consistent on the interface with PP?

Reply:
here is some input of MOE_Layer for recompute. pp layer will universally be modified according to competing product in the future.